container-chains/templates/frontier/node/src/command.rs

```diff
@@ -360,10 +360,14 @@ pub fn run() -> Result<()> {
 				let parachain_account =
 					AccountIdConversion::<polkadot_primitives::AccountId>::into_account_truncating(&id);
 
-                // TODO: modify
-				let block: Block = generate_genesis_block(&*config.chain_spec, sp_runtime::StateVersion::V1)
+                // We log both genesis states for reference, as fetching it from runtime would take significant timw
```

Comment:
Not sure how useful is logging both, but it's amazing that generating 2 states is faster than fetching it from the runtime.

```suggestion
                // We log both genesis states for reference, as fetching it from runtime would take significant time
```